docs/application/native/guides/connectivity/inm.md

```diff
@@ -0,0 +1,171 @@
+# INM
+
+
+INM means Intelligent Network Monitoring. It can be used to get information about the network provided by well-known linux functions. Some features provided by this API are: detecting IP collision, dumping network status, dumping TCP, monitoring ethernet, monitoring Wi-Fi module state and getting network statistics.
+
+This feature is supported in mobile, tv and wearable profile.
+
+> **Note**  
+> You can test the INM functionality on a target device only. The [emulator](../../../tizen-studio/common-tools/emulator.md) does not support this feature.
```

Comment:
Kindly use the [updated Note format](https://github.com/Samsung/tizen-docs/blob/master/styleguide/custom-style.md#alerts-note-tip-important-caution-warning) and also tweaking the sentence as:

\> [!NOTE]
\> You can test the INM functionality on a target device only. The [Tizen emulator](../../../tizen-studio/common-tools/emulator.md) does not support this feature.

Kindly verify for the technical accuracy.

Reply:
Waiting for the staging build to see if the custom markup looks correct.